docs/sources/datasources/azure-monitor/_index.md

```diff
@@ -140,7 +140,7 @@ For details on Azure managed identities, refer to the [Azure documentation](http
 
 ## Query the data source
 
-The Azure Monitor data source can query data from Azure Monitor Metrics and Logs, and the Azure Resource Graph, each with its own specialized query editor.
+The Azure Monitor data source can query data from Azure Monitor Metrics and Logs, the Azure Resource Graph, and Application Insights Traces each with its own specialized query editor.
```

Comment:
The phrasing for "each with its own specialized query editor" sounds awkward, so I'm suggesting separating it into another sentence. Is the wording below correct? What's the correct term for "Azure Monitor Metrics and Logs, Azure Resource Graph, and Application Insights Traces"? Are they sources or something else? 
```suggestion
The Azure Monitor data source can query data from Azure Monitor Metrics and Logs, the Azure Resource Graph, and Application Insights Traces. Each source has its own specialized query editor.
```